qutip/tests/solver/test_parallel.py

```diff
@@ -22,14 +22,17 @@ def _func2(x, a, b, c, d=0, e=0, f=0):
 @pytest.mark.parametrize('map', [
     pytest.param(parallel_map, id='parallel_map'),
     pytest.param(loky_pmap, id='loky_pmap'),
+    pytest.param(mpi_pmap, id='mpi_pmap'),
     pytest.param(serial_map, id='serial_map'),
 ])
 @pytest.mark.parametrize('num_cpus',
                          [1, 2],
                          ids=['1', '2'])
 def test_map(map, num_cpus):
     if map is loky_pmap:
-        loky = pytest.importorskip("loky")
+        pytest.importorskip("loky")
+    if map is mpi_pmap:
+        pytest.importorskip("mpi4py")
```

Comment:
Will `2` cpus check be reliable with mpi?

Reply:
I think so. With setting `OMPI_MCA_rmaps_base_oversubscribe=true`, there should be no limit to the number of processes.

The only potential issue I can see is that the name of that environment variable was changed in OpenMPI 5. However, currently mpi4py only supports OpenMPI up to 4. Therefore I pinned the OpenMPI version to "<5".